src/sentry/monitors/validators.py

```diff
@@ -257,8 +256,25 @@ class MonitorValidator(CamelSnakeSerializer):
     config = ConfigValidator()
     alert_rule = MonitorAlertRuleValidator(required=False)
 
-    def validate_status(self, value):
-        return MONITOR_STATUSES.get(value, value)
+    def validate(self, attrs):
+        status = attrs.get("status")
+
+        # TODO(epurkhiser): This translation of muted / active status ->
+        # isMuted will be removed in the future.
+        #
+        # When the status is set to muted we will set is_muted true, the status
+        # will not change. When the status is set to 'active' we will set
+        # is_muted to false and the status will be set to active
+        if status == "active":
+            attrs["is_muted"] = False
+
+        if status == "muted":
+            del attrs["status"]
+            attrs["is_muted"] = True
+        else:
+            attrs["status"] = MONITOR_STATUSES.get(status)
```

Comment:
Here's the bit that implements translation of legacy "muted" status to `is_muted`